Makefile

```diff
@@ -915,6 +915,13 @@ generate-manpage: ## generate manpage
 	@gzip -9 man/man1/gitea.1 && echo man/man1/gitea.1.gz created
 	@#TODO A small script that formats config-cheat-sheet.en-us.md nicely for use as a config man page
 
+.PHONY: generate-completions
+generate-completions: ## generate shell completions
+	@[ -f gitea ]  || make backend
+	@./gitea completion bash > contrib/autocompletion/bash_autocomplete
+	@./gitea completion zsh  > contrib/autocompletion/zsh_autocomplete
+	@./gitea completion fish > contrib/autocompletion/gitea.fish
+
```

Comment:
When end users need the "autocompletion", they must have had the "gitea" binary on their filesystem.

So I think it's better to remove `contrib/autocompletion/*`, just tell end users to generate their autocompletion scripts by running the command.

Then we do not need to maintain these scripts in this repo.